.travis.yml

```diff
@@ -33,7 +33,7 @@ install:
   - if [ "$DEPENDENCIES" == "low" ]; then composer update --prefer-lowest; fi;
 
 before_script:
-  - echo "<?php if (PHP_VERSION_ID >= 50400) echo ',@php5.4';" > php_version_tags.php
```

Comment:
Yeah it'll be ok for now. I'll probably write a behat extension or something that does this better